client/src/components/statCalculations/AssignDisplay.js

```diff
@@ -0,0 +1,96 @@
+import styled from "styled-components";
+import React, { Component, createRef } from "react";
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 15vh;
+  display: flex;
+  justify-content: center;
+`;
+export default class AssignDisplay extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // statIndex tracks the index of what stat we are setting, its always one ahead
+      statIndex: 0,
+      buttonsDisabled: [false, false, false, false, false, false]
+    };
+  }
+
```

Comment:
Line 16: button disabled here is ok but in the future should think of a more elegant solution than 6 indexes of false.

Reply:
I know it looks not elegant, but at the moment there needs to be a way to keep track of which buttons are disabled so I can reset it with the clear button

Now, the button thing doesn't have to be exactly how this is done, just an initial POC